src/api/LuaApiProvider.tsx

```diff
@@ -30,7 +32,11 @@ export function LuaApiProvider({ children }: PropsWithChildren) {
         const res = await api.singleReturnLibrary();
         setLuaApi(res);
       } catch (e) {
-        console.error(e);
+        if (isReactRenderable(e)) {
+          showNotification('Error fetching OpenSpace API', e, LogLevel.Error);
```

Comment:
Wait, what? is the thing we're checking here actually just if we got an error?  Could we change the check to make that more clear? I.e. maybe flip the if statement somehow

Reply:
The check is if we can show the error as a notification, since `e`is unkown from the catch we must verify its type as a React.Node before using `showNotification` otherwise we will just print it to the console